invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -70,12 +70,18 @@ class AccessSettingsSchema(Schema):
     allow_user_requests = fields.Boolean()
     allow_guest_requests = fields.Boolean()
 
-    accept_conditions_text = SanitizedHTML()
+    accept_conditions_text = SanitizedHTML(allow_none=True)
+    secret_link_expiration = fields.Integer(allow_none=True)
```

Comment:
Shall we put a `max limit`? E.g. 1 year? Or can I overflow the max integer? :D

Reply:
my integer? :woozy_face: